### PR TITLE
Add check and test for empty isochrone.

### PR DESCRIPTION
**Pull request by flibbertigibbet**
Unlikely to occur, but may happen if origin outside graph bounds, or inaccessible.

See #67 comments.

### CODE REVIEW

File: python/cac_tripplanner/destinations/views.py [hunk 1]
```diff
@@ -45,7 +45,11 @@ def isochrone(self, lat, lng, mode, date, time, max_travel_time, max_walk_distan
         isochrone_response = requests.get(self.isochrone_url, params=payload)
 
         # Parse and traverse JSON from OTP so that we return only geometries
-        json_poly = json.loads(isochrone_response.content)[0]['geometry']['geometries']
+        try:
+            json_poly = json.loads(isochrone_response.content)[0]['geometry']['geometries']
```
Comment (notthatbreezy) [inline python/cac_tripplanner/destinations/views.py]: Requests has a nice helper method you can use to save the call to json.loads, `isochrone_response.json()`

**Comment by notthatbreezy** [comment]
+1

File: python/cac_tripplanner/destinations/views.py (hunk 1 above)
Comment (flibbertigibbet) [inline python/cac_tripplanner/destinations/views.py]: Oh, we have requests in the project.  I saw the feed parser was just using urllib.

Comment (notthatbreezy) [inline python/cac_tripplanner/destinations/views.py]: I haven't looked at the feed parser, but at least here we're using requests https://github.com/azavea/cac-tripplanner/blob/2a90ebd68885efc0549988167223a2c0a45bb6a7/python/cac_tripplanner/destinations/views.py#L45
